sample/src/main/java/org/aerogear/graphqlandroid/activities/MainActivity.kt

```diff
@@ -68,9 +92,16 @@ class MainActivity : AppCompatActivity() {
 
         Log.e(TAG, "inside getTasks")
 
+
         val client = Utils.getApolloClient(this)?.query(
             AllTasksQuery.builder().build()
-        )
+        )?.httpCachePolicy(HttpCachePolicy.CACHE_FIRST)
```

Comment:
This means that you will always get oudated data as result (cache will be refereshed asynchronously). This model requires cache watch method connected to UI. Everytime something changes in cache UI should be refreshed, but I do not see it implemented here. 

To gather understanding of the process implement swipe down refresh on list or something that will let you manually query data.